XCUITests/SettingsTest.swift

```diff
@@ -41,7 +41,7 @@ class SettingsTest: BaseTestCase {
             // Tap on "Default Browser App" and set the browser as a default (Safari is listed first)
             waitForExistence(iOS_Settings.tables.cells.element(boundBy: 1), timeout: 5)
             iOS_Settings.tables.cells.element(boundBy: 2).tap()
-            iOS_Settings.tables.staticTexts.element(boundBy: 1).tap()
+            iOS_Settings.tables.staticTexts.element(boundBy: 0).tap()
```

Comment:
I know this is how we did in the past...but, any change we can do this in a better way so that we don't have to change that again in the future...something using the name or similar... just if you have time or feel like investigating a bit about this... 
Thanks!

Reply:
we can use app.label to store app name and use it to select the app, so it won't fail in case of an order change.